locales/de.json

```diff
@@ -19,6 +34,60 @@
 			"startM2018": "(Start Mai 2018)"
 		}
 	},
+	"courses": {
+		"course": {
```

Comment:
I have updated the guidelines to prefix dynamic url parts (id's) with an underscore. That should make it easier for us to identify those and not get crazy because we can not find `/courses/course`.
I will update this in your PR, because it was my fault to have forgotten this part in the documentation.
![image](https://user-images.githubusercontent.com/22987140/76829916-85bba500-6824-11ea-81c9-31d7e1baad1c.png)